enterprise_learner_portal/utils.py

```diff
@@ -43,7 +43,9 @@ def get_course_run_status(course_overview, certificate_info, enterprise_enrollme
     is_certificate_passing = certificate_info.get('is_passing', False)
     certificate_creation_date = certificate_info.get('created', datetime.max)
 
-    if enterprise_enrollment and enterprise_enrollment.marked_done:
+    if enterprise_enrollment and enterprise_enrollment.marked_done is False:
```

Comment:
I am going to pull this change out since it breaks other areas. I think we may need to rethink the course status returned to the portal rather than change this logic ...